### DISCUSSION TITLE
Weather Forecast API: Hourly variable "Visibility"

### DISCUSSION THREAD

**Discussion by hoetzgit** [open-meteo/open-meteo#234] [Q&A] [answered]
@patrick-zippenfenig

Hello Patrick, the data field `Visibility` in the hourly variables is not available for every model in the endpoint `v1/forecast` (e.g. DWD *). Is this correct or a bug? If it is not a bug, maybe a note in the documentation would be useful?

Many thanks for your great work.

Regards
Henry

**Comment by patrick-zippenfenig** [comment, marked as answer]
Correct. `Visibility` is only available by the American weather models.

To better document all available data, I create dedicated API documentations for ICON, GFS, MeteoFrance, etc. In general it is a huge mess with all those models, different variables, resolutions, forecast lengths, etc.

A table with weather variable and model could be an option to better document it.
